api/rbdigital.py

```diff
@@ -2616,7 +2648,6 @@ def import_spine(self, file_data, proxy_url):
         extra = {}
         extra['proxy_link'] = dict(
             href=proxy_url,
-            type=Representation.guess_media_type(filename)
```

Comment:
Why take this out?

Reply:
We are no longer returning an `alternate` URL, but a proxy link. The clients are expecting to see `vnd.librarysimplified/rbdigital-access-document+json` for an RBdigital access document, which is what the proxy would return. `Representation.guess_media_type(filename)` would return `audio/mpeg`, which would lead the clients to try to download and stash an access document as the content, rather than using it as a pointer to the content.

Reply:
That makes sense, thanks.